sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -545,8 +544,8 @@ public abstract partial class RequestContent : System.IDisposable
         public static Azure.Core.RequestContent Create(byte[] bytes, int index, int length) { throw null; }
         public static Azure.Core.RequestContent Create(System.IO.Stream stream) { throw null; }
         public static Azure.Core.RequestContent Create(object serializable) { throw null; }
+        public static Azure.Core.RequestContent Create(object serializable, Azure.Core.Serialization.JsonPropertyNames propertyNameFormat, string dateTimeFormat = "o") { throw null; }
```

Comment:
Should it be JsonPropertyNameConventions?

Reply:
Yeah, I think you're right.  I liked the suggestion you made as well to just rename the property.  Let me do that real quick, we'll see what it looks like.

Reply:
I think JsonPropertyNames means "these are the names in JSON payload". It does not say anything about conversions. If we call it JsonPropertyNameConvention, it will be strange to see JsonPropertyNameConvention.CamelCase to mean that now you can use PascalCase (when reading). This is at the root of the confusion that Ted had a problem with

Reply:
@KrzysztofCwalina, I've put it back to PropertyNameFormat, but renamed the parameter to `jsonPropertyNameFormat` - how do you feel about this modification?

Reply:
It means we could use the same enum with other data types in the future, without needing to create new enums for each one, but still gets the benefit of designating in the parameter name that you are talking about the data.

Reply:
Hopefully, regarding the original point on this comment, keeping the parameter name as `propertyNameFormat` will help clarify the vagueness.